Pod/Classes/XExtensionItem.h

```diff
@@ -87,13 +87,14 @@
 @property (nonatomic, copy) NSDictionary *userInfo;
 
 /**
- Initialize a new instance with a placeholder item, whose type will be used by the activity controller to determine 
+ Initialize a new instance with a placeholder item, whose type will be used by the activity controller to determine
  which activities and extensions are displayed, and an array of attachments that will actually be passed to the selected 
  activity/extension.
  
- @param placeholderItem (Required) A placeholder item whose type will be used by the activity controller to determine
- which activities and extensions are displayed.
- @param attachments     (Required) An array of media data associated with the extension item. These items must be of 
+ @param placeholderItem A placeholder item whose type will be used by the activity controller to determine
+ which activities and extensions are displayed. System activities that do not accept `NSExtensionItem` input will 
+ receive this placeholder instead.
```

Comment:
> System activities that do not accept `NSExtensionItem` input will receive this placeholder instead.

Should we / do we handle the case where I want to actually use the placeholder as it’s intended, to be an object of the same type, but not the final object, and provide the object later?